src/HTMLElement/elements/HTMLTableElement.js

```diff
@@ -0,0 +1,322 @@
+import HTMLElement from '../../HTMLElement';
+
+/**
+ * The HTMLTable interface represents table
+ * and inherits all classes and methods of the HTMLElement interface.
+ *
+ * @see https://developer.mozilla.org/en/docs/Web/API/HTMLTableElement
+ * @class HTMLTableElement
+ */
+export default class HTMLTableElement extends HTMLElement {
+    constructor() {
+        super();
+    }
+
+    /**
+     * Caption of element, returns HTMLElement <caption>
+     * @member {HTMLElement} HTMLTableElement#caption
+     */
+    get caption() {
+        return this.querySelector('caption');
+    }
+
+    /**
+     * Sets caption of element
+     * @param {HTMLElement} element element with tagName caption
+     * @member {HTMLElement} HTMLTableElement#caption
+     */
+    set caption(element) {
+        if (element.tagName !== 'caption') {
+            throw new DOMException('HierarchyRequestError');
+        }
+
+        let caption = this.querySelector('caption');
+        if (caption !== null) {
+            this.insertBefore(element, caption);
+            this.removeChild(caption);
+        } else {
+            if (this.firstChild) {
+                this.insertBefore(element, this.firstChild);
+            } else {
+                this.appendChild(element);
+            }
+        }
+    }
+
+    /**
+     * Table head
+     * @member {HTMLElement} HTMLTableElement#tHead
+     */
+    get tHead() {
+        return this.querySelector('thead');
```

Comment:
According to the documentation, it is `the first <thead> that is a child of the element, or null if none is found`, so you can do instead `this.children.find(child => child.tagName === 'thead') || null`, and modify the jsdoc to `@member {HTMLElement|null} HTMLTableElement#tHead`

I'm not sure that Array.prototype.find is implemented in node, but we could add a method `_childNodeFind` like https://github.com/christophehurpeau/html-document/blob/master/src/ParentNode.js#L35

```js
    _childNodeFind(callback) {
        let result = null;
        this._childNodes.some(node => {
            if (callback(node)) {
                result = node;
                return true;
            }
        });
        return result;
    }
```

Then you can use: `this._childNodeFind(child => child.tagName === 'thead')`